x-pack/plugins/ml/public/components/annotations/annotation_flyout/index.tsx

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, ComponentType, Fragment, ReactNode } from 'react';
+import * as Rx from 'rxjs';
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiFormRow,
+  EuiSpacer,
+  EuiTextArea,
+  EuiTitle,
+} from '@elastic/eui';
+
+import {
+  annotation$,
+  annotationsRefresh$,
+  AnnotationState,
+} from '../../../services/annotations_service';
+import { injectObservablesAsProps } from '../../../util/observable_utils';
+import { AnnotationDescriptionList } from '../annotation_description_list';
+import { DeleteAnnotationModal } from '../delete_annotation_modal';
+
+import { CommonProps } from '@elastic/eui';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+import { InjectedIntlProps } from 'react-intl';
+import { toastNotifications } from 'ui/notify';
+
+import { ml } from '../../../services/ml_api_service';
+
+interface Props {
+  annotation: AnnotationState;
+}
+
+interface State {
+  isDeleteModalVisible: boolean;
+}
+
+class AnnotationFlyoutIntl extends Component<CommonProps & Props & InjectedIntlProps> {
+  public state: State = {
+    isDeleteModalVisible: false,
+  };
+
+  public annotationSub: Rx.Subscription | null = null;
+
+  public annotationTextChangeHandler = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    if (this.props.annotation === null) {
+      return;
+    }
+
+    annotation$.next({
+      ...this.props.annotation,
+      annotation: e.target.value,
+    });
+  };
+
+  public cancelEditingHandler = () => {
+    annotation$.next(null);
+  };
+
+  public deleteConfirmHandler = () => {
+    this.setState({ isDeleteModalVisible: true });
+  };
+
+  public deleteHandler = async () => {
+    const { annotation, intl } = this.props;
+
+    if (annotation === null) {
+      return;
+    }
+
+    try {
+      await ml.annotations.deleteAnnotation(annotation._id);
+      toastNotifications.addSuccess(
+        intl.formatMessage(
+          {
+            id: 'xpack.ml.timeSeriesExplorer.timeSeriesChart.deletedAnnotationNotificationMessage',
+            defaultMessage: 'Deleted annotation for job with ID {jobId}.',
+          },
+          { jobId: annotation.job_id }
+        )
+      );
+    } catch (err) {
+      toastNotifications.addDanger(
+        intl.formatMessage(
+          {
+            id:
+              'xpack.ml.timeSeriesExplorer.timeSeriesChart.errorWithDeletingAnnotationNotificationErrorMessage',
+            defaultMessage:
+              'An error occured deleting the annotation for job with ID {jobId}: {error}',
```

Comment:
Typo - should be `occurred`

Reply:
Fixed in this separate PR: https://github.com/elastic/kibana/pull/30102